### PR TITLE
added {USER} replacement

### PR DESCRIPTION
**Pull request by fceller**
backport of https://github.com/arangodb/arangodb/pull/13434

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v3.7.7 (XXXX-XX-XX)
 -------------------
 
+* Allow {} paceholder string also in `--ldap.search-filter`.
```
Comment (KVS85) [inline CHANGELOG:4]: * Allow {USER} paceholder string also in `--ldap.search-filter`.

**Review by KVS85 — APPROVED** [review]
LGTM.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13715/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13718/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13718/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13720/

**Review by joerg84 — APPROVED** [review]

**Comment by KVS85** [comment]
Tests blue.

**Review by jsteemann — APPROVED** [review]

**Pull request merged**